src/main/views/crown-firm-list.njk

```diff
@@ -0,0 +1,171 @@
+    <p class="govuk-body">{{ draft }} {{ versionText }} {{ version }}</p><br/>
+    <p class="govuk-body">
+      {% for line in listData['venue']['venueAddress']['line'] %}
+        {{ line }}<br/>
+      {% endfor %}
+      {% if listData['venue']['venueAddress']['town'] | length %}
+        {{ listData['venue']['venueAddress']['town'] }}<br/>
+      {% endif %}
+      {% if listData['venue']['venueAddress']['county'] | length %}
+        {{ listData['venue']['venueAddress']['county'] }}<br/>
+      {% endif %}
+      {{ listData['venue']['venueAddress']['postCode'] }}
+    </p>
+    <div class="govuk-grid restriction-list-section">
+      <div class="govuk-body">
+        <h3>{{ restrictionInformationHeading }}</h3>
+        <p class="govuk-body">{{ restrictionInformationP1 }}</p>
+        <div class="govuk-warning-text">
+          <span class="govuk-warning-text__icon align-warning-icon" aria-hidden="true">!</span>
+          <strong class="govuk-warning-text__text">
+            <span class="govuk-warning-text__assistive">Warning</span>
+            {{ restrictionInformationBoldText }}
+          </strong>
+        </div>
+        <p class="govuk-body">{{ restrictionInformationP2 }}</p>
+        <p class="govuk-body">{{ restrictionInformationP3 }}</p>
+        <p class="govuk-body">{{ restrictionInformationP4 }}</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>{{ restrictionBulletPoint1 }}</li>
+          <li>{{ restrictionBulletPoint2 }}</li>
+        </ul>
+        <p class="govuk-body">{{ restrictionInformationP5 }} <a href="#" class="govuk-link">{{ restrictionLink }}</a>
+        </p>
+      </div>
+      {% if bill %}
+        <div class="govuk-body">
+          <h3>{{ restrictionInformationHeadingBill }}</h3>
+          <p class="govuk-body">{{ restrictionInformationP1Bill }}</p>
+          <div class="govuk-warning-text">
+            <span class="govuk-warning-text__icon align-warning-icon" aria-hidden="true">!</span>
+            <strong class="govuk-warning-text__text">
+              <span class="govuk-warning-text__assistive">Warning</span>
+              {{ restrictionInformationBoldTextBill }}
+            </strong>
+          </div>
+          <p class="govuk-body">{{ restrictionInformationP2Bill }}</p>
+          <p class="govuk-body">{{ restrictionInformationP3Bill }}</p>
+          <p class="govuk-body">{{ restrictionInformationP4Bill }}</p>
+          <ul class="govuk-list govuk-list--bullet">
+            <li>{{ restrictionBulletPoint1Bill }}</li>
+            <li>{{ restrictionBulletPoint2Bill }}</li>
+          </ul>
+          <p class="govuk-body">{{ restrictionInformationP5Bill }} <a href="#"
+                                                                      class="govuk-link">{{ restrictionLinkBill }}</a>
+          </p>
+        </div>
+      {% endif %}
+    </div>
+  </div>
+  {% for court in allocated %}
```

Comment:
Only super minor, but there's a few formatting differences below (some blocks have spaces before, some don't etc)

Could this be updated to make it consistent?

Reply:
sorted I think?